vagrant/provisioning/roles/pki_client/tasks/main.yml

```diff
@@ -19,65 +19,52 @@
   command: openssl rsa -in {{ ssl_key }} -out {{ ssl_key_rsa }} -passin pass:'{{ ssl_key_passphrase }}'
   when: not ssl_key_rsa_stat.stat.exists|bool and ssl_key_passphrase is defined
 
-- name: ensure secure permissions on private keys
+- name: see if the Java keystore exists
```

Comment:
In FIPS mode, the OpenJDK keytool utility doesn't work.  Need to get all services to use PEM or pkcs12 key stores.